op-node/rollup/derive/interop_upgrade_transactions.go

```diff
@@ -0,0 +1,102 @@
+)
+
+func InteropNetworkUpgradeTransactions() ([]hexutil.Bytes, error) {
+	upgradeTxns := make([]hexutil.Bytes, 0, 4) // 2 + 2 updates
+
+	// 1. Deploy CrossL2Inbox
+	deployCrossL2InboxTx, err := types.NewTx(&types.DepositTx{
+		SourceHash:          deployCrossL2InboxSource.SourceHash(),
+		From:                crossL2InboxDeployerAddress,
+		To:                  nil, // Contract creation
+		Mint:                big.NewInt(0),
+		Value:               big.NewInt(0),
+		Gas:                 420000,
```

Comment:
```suggestion
		Gas:                 420_000,
```
nit, let's separate by thousands here as well